Index.html

```diff
@@ -8,6 +8,10 @@
 <body>
   <header>
     <nav id="navBar">
+      <div class="nav__bgPicture">
+          <img class="nav__pPicture" src="images\miguel_profilePicture.jpg" alt="Profile Picture">
```

Comment:
You could be more specific with the alternative text `Miguel Solano's profile picture`